train_segmentation.py

```diff
@@ -427,16 +446,6 @@ def main(params, config_path):
     bucket_name = get_key_def('bucket_name', params['global'])
     data_path = Path(params['global']['data_path'])
     assert data_path.is_dir(), f'Could not locate data path {data_path}'
-<<<<<<< HEAD
```

Comment:
Please, check on these spaces that may have been created across all the files changed

Reply:
same as the other one